packages/manager/apps/telecom/src/app/telecom/telephony/billingAccount/telecom-telephony-billing-account.html

```diff
@@ -20,62 +20,49 @@
-                        <span data-translate="telephony_group_tab_guides"></span>
-                    </responsive-tab>
-
-                    <responsive-tab-more>
-                        <i class="fa fa-plus"></i>
-                    </responsive-tab-more>
-                </responsive-tabs>
+                <oui-header-tabs>
+                    <oui-header-tabs-item
+                        data-text="{{ 'telephony_group_tab_dashboard' | translate }}"
+                        data-state="telecom.telephony.billingAccount">
+                    </oui-header-tabs-item>
+                    <oui-header-tabs-item
+                        data-ng-if="BillingAccountCtrl.isBetaActive"
+                        data-text="{{ 'telephony_group_tab_services' | translate }}"
+                        data-state="telecom.telephony.billingAccount.services">
+                    </oui-header-tabs-item>
+                    <oui-header-tabs-item
+                        data-text="{{ 'telephony_group_tab_admin' | translate }}"
+                        data-state="telecom.telephony.billingAccount.administration">
+                    </oui-header-tabs-item>
+                    <oui-header-tabs-item
+                        data-text="{{ 'telephony_group_tab_billing' | translate }}"
+                        data-state="telecom.telephony.billingAccount.billing">
+                    </oui-header-tabs-item>
+                    <oui-header-tabs-item
+                        data-text="{{ 'telephony_group_tab_orderAlias' | translate }}"
+                        data-state="telecom.telephony.billingAccount.orderAlias"
+                        data-disabled="!BillingAccountCtrl.canOrderAlias">
+                    </oui-header-tabs-item>
+                    <oui-header-tabs-item
+                        data-text="{{ 'telephony_group_tab_contacts_book' | translate }}"
+                        data-state="telecom.telephony.billingAccount.phonebook"
+                        data-disabled="BillingAccountCtrl.loading.init">
+                    </oui-header-tabs-item>
+                    <oui-header-tabs-item
+                        data-text="{{ 'telephony_group_tab_short_numbers' | translate }}"
+                        data-state="telecom.telephony.billingAccount.abbreviatedNumbers"
+                        data-disabled="BillingAccountCtrl.loading.init">
+                    </oui-header-tabs-item>
+                    <oui-header-tabs-item
+                        data-text="{{ 'telephony_group_tab_contacts_manage' | translate }}"
+                        data-state="telecom.telephony.billingAccount.manageContacts"
+                        data-disabled="BillingAccountCtrl.loading.init">
+                    </oui-header-tabs-item>
+                    <oui-header-tabs-item
+                        data-text="{{ 'telephony_group_tab_guides' | translate }}"
```

Comment:
Existing a specific reason about removing the one-time binding?

Reply:
No reason, I probably removed it without being careful